network/datadog_checks/network/network.py

```diff
@@ -67,6 +68,263 @@
     "pps_allowance_exceeded",
 ]
 
+ETHTOOL_METRIC_NAMES = {
```

Comment:
I think this, along with the other consts should be move to a `const.py` file

Reply:
Done